src/web/components/Subtitle/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Subtitle({ children, level }) {
+  const classes = `subtitle${` is-${level + 1}`}`;
+  const Heading = `h${level + 1}`;
```

Comment:
```javascript
level + 1
```

is made two times inside this component. Maybe we could make it only once ?

Reply:
Is this working ?

Reply:
Yes, it works :smile: .
Initially I used a switch case for the tests, but I couldn't get 100% coverage without doing a test per case. So I did a little research on Stackoverflow and I found this solution.

I'm gonna refactor level.